test/SparqlEndpointFetcher-test.ts

```diff
@@ -175,14 +175,18 @@ describe('SparqlEndpointFetcher', () => {
     describe('#fetchRawStream', () => {
       it('should pass the correct URL and HTTP headers', () => {
```

Comment:
I would suggest splitting this test up into:
1. 'should pass the correct URL and HTTP headers' (=the original test)
2. 'should pass the correct URL and HTTP headers with additional URL parameters' (=the test you have here)

Reply:
Done